Client/Blogger/src/Components/NavBar.jsx

```diff
@@ -0,0 +1,45 @@
+import Container from "react-bootstrap/Container";
+import { Nav, Navbar,Button } from "react-bootstrap";
+import { NavLink } from "react-router-dom";
+import { useSelector, useDispatch } from "react-redux";
+import CountryList from './DropDown'
+function NavigationBar() {
+  
+  const { isLoggedIn } = useSelector((state) => state.user);
+  console.log("@logged-in",isLoggedIn)
```

Comment:
avoid leaving unnecessary console logs in the code while commit, only keep if it's important for debugging or for the app's functionality